android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -250,7 +255,7 @@ public void notifyConnection(@Nullable VehicleType vehicleType) {
         }
     }
 
-    private static void saveVehicleType(Context context, VehicleType vehicleType, String address) {
+    public static void saveVehicleType(Context context, VehicleType vehicleType, String address) {
```

Comment:
@kostyaBoss On line 258 why was this method changed to `public`?  `saveVehicleType` is not being invoked from anywhere else.